pandas/core/series.py

```diff
@@ -2462,7 +2462,7 @@ def searchsorted(self, value, side="left", sorter=None):
     # -------------------------------------------------------------------
     # Combination
 
-    def append(self, to_append, ignore_index=False, verify_integrity=False) -> "Series":
```

Comment:
rather than deleting the annotation, can you use FrameOrSeriesUnion from pandas._typing instead.

Reply:
@simonjayhawkins 
FrameOrSeriesUnion is throwing this error during static analysis:
**Performing static analysis using mypy
pandas/core/frame.py:2590: error: Incompatible types in assignment (expression has type "Union[DataFrame, Series]", variable has type "Series")**

And FrameOrSeries is throwing this:
**Performing static analysis using mypy
pandas/core/series.py:2505: error: Incompatible return value type (got "Union[DataFrame, Series]", expected "FrameOrSeries")**

I guess this is why there was no type hint initially for 0.25.3 version of append.
What do you think about this?

Thanks!

Reply:
OK perhaps we should keep this PR to just fixing the regression. so removing the return annotation is fine here.